logpuzzle.py

```diff
@@ -40,14 +75,25 @@ def download_images(img_urls, dest_dir):
     with an img tag to show each local image file.
     Creates the directory if necessary.
     """
-    # +++your code here+++
-    pass
+
+    os.mkdir(dest_dir)
+
+    with open('index.html', 'w') as index:
```

Comment:
The index.html file handle is open while all downloads complete. Consider waiting until all downloads have completed before opening the index.html file handle.